app/models/plan.rb

```
@@ -24,6 +26,8 @@ class Plan < ApplicationRecord
   validates :code, presence: true, uniqueness: { scope: :organization_id }
   validates :amount_currency, inclusion: { in: currency_list }
 
+  scope :default, -> { where(overridden_plan_id: nil) }
```

`default_scope` instead?

---

I can switch to default scope but I personally don't like it since it mask behaviour :/

---

I tend to agree 😅 

But I think we can use a different name than `default` then, something more explicit?

---

`root`? 😃

---

`parent` makes more sense I think.

`default` could means many many things :smile:

---

what about `base` ?

---

switched to `base`

---

`not_overriden`, explicit 😅